File: package-protocol-hack/src/main/java/de/florianmichael/clampclient/injection/signature/provider/CommandArgumentsProvider.java
package de.florianmichael.clampclient.injection.signature.provider;

import com.viaversion.viaversion.api.platform.providers.Provider;
import net.minecraft.util.Pair;

import java.util.List;

public class CommandArgumentsProvider implements Provider {

    public List<Pair<String, String>> getSignedArguments(final String command) {
        return null;
    }
}

File: package-protocol-hack/src/main/kotlin/de/florianmichael/tarasande_protocol_hack/fix/chatsession/v1_19_2/CommandArgumentsProvider.kt
package de.florianmichael.tarasande_protocol_hack.fix.chatsession.v1_19_2

import com.viaversion.viaversion.api.platform.providers.Provider

open class CommandArgumentsProvider : Provider {

    open fun getSignedArguments(command: String): List<Pair<String, String>> {
        return listOf()
    }
}
